runtime/onert/backend/gpu_cl/ClFunction.h

```diff
@@ -18,6 +18,8 @@
 #define __ONERT_GPU_CL_CL_FUNCTION_H__
```

Comment:
on other header files on runtime backend includes directory name on define.
how about apply it on this file too?

Reply:
Modified.